support-frontend/assets/helpers/subscriptionsForms/formFields.js

```diff
@@ -51,6 +53,7 @@ export type FormState = {|
   isTestUser: boolean,
   productPrices: ProductPrices,
   payPalHasLoaded: boolean,
+  orderIsAGift: boolean,
```

Comment:
There's something a bit odd going on here because we've already defined `orderIsAGift` in the `FormFields` type above which is spread into this type.

I think you can remove `orderIsAGift` here and make the definition in `FormField` non optional ie.
```
orderIsAGift: boolean,
```

Reply:
Oh noes. Will fix. Thanks Rupert!